modules/backend/layouts/_head.htm

```diff
@@ -50,6 +50,8 @@
     <link href="<?= $script . '?v=' . $coreBuild; ?>" rel="preload" as="script" importance="high">
 <?php endforeach; ?>
 
+    <link href="<?php Url::asset('modules/system/assets/ui/font/fontawesome-webfont.woff?v=1.0.1'); ?>" rel="preload" as="font" type="font/woff" crossorigin importance="high">
```

Comment:
this shouldn't be indented here, the other ones are indented because they're in a PHP code block.